proto/internalapi/sensor/compliance_iservice.proto

```diff
@@ -51,11 +51,19 @@ message MsgToCompliance {
             StopRequest  stop_req  = 2;
         }
     }
```

Comment:
Add a comment here that the Nack will be removed and that's why its not consolidated with the Ack

Reply:
Should we have this as a //TODO(ROX-xxxx) item with a follow up so we don't forget about it?